Storage/StorageTests/CoreData/CoreDataManagerTests.swift

```diff
@@ -35,14 +36,14 @@ final class CoreDataManagerTests: XCTestCase {
     /// Verifies that the ContextManager's viewContext matches the PersistenContainer.viewContext
     ///
     func test_viewContext_property_returns_persistentContainer_main_context() {
-        let manager = CoreDataManager(name: "WooCommerce", crashLogger: MockCrashLogger())
+        let manager = CoreDataManager(name: storageIdentifier, crashLogger: MockCrashLogger())
```

Comment:
I was going to comment "would you consider extracting this to a private method" but, to be honest, it's just a line, I am not sure it adds any value, other than the semantic value of calling it `createDefaultManager()` or similar, so maybe it's not worth it.

Sorry, I' just thinking out loud here...

Reply:
I thought that too and we both ended up with the same conclusion. 😄